releasenotes/notes/deprecate-datadog-env-3011aafe83b66e37.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    Deprecate the DATADOG_ENV environment variable in favor of DD_ENV . 
+    The use of DD_ENV should follow Unified Service Tagging
+    recommendations.
```

Comment:
do we have a link we can share to these recommendations?

Not sure "Unified Service Tagging" is going to mean much to most people.

Reply:
Does reno support markdown to add links? I couldn't find it in the examples here: https://docs.openstack.org/reno/latest/user/examples.html .

I was thinking of adding a link to https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/ specifically, but wasn't sure how to format it.

Reply:
it is RST not Markdown.

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

You can also validate the rendering in the `build_docs` CircleCI artifacts.

e.g. https://845293-61572326-gh.circle-artifacts.com/0/tmp/docs/release_notes.html

Reply:
That linked worked, latest artifact shows the link to the public docs on Unified Service Tagging: https://845704-61572326-gh.circle-artifacts.com/0/tmp/docs/release_notes.html ! 👍